ios/FluentUI/Drawer/DrawerPresentationController.swift

```diff
@@ -77,12 +77,15 @@ class DrawerPresentationController: UIPresentationController {
         // Pass the passthrough view in touch forwarding view
         if let passThroughView = self.passThroughView {
             view.passthroughView = passThroughView
+            view.accessibilityElements = [passThroughView]
```

Comment:
Should passthroughview even be part of the accessibilityelements?

Reply:
@harrieshin Lets suppose we have passthroughView as the multi-select grid view. In that case through voice over user will not be able to access the passthroughView. After adding it to accessibilityElements, it can be accessed.